armory/art_experimental/attacks/carla_obj_det_utils.py

```diff
@@ -97,3 +106,208 @@ def rgb_depth_to_linear(r, g, b):
     depth_m = r_ + g_ * 256 + b_ * 256 * 256
     depth_m = depth_m * 1000.0 / (256**3 - 1)
     return depth_m
+
+
+@dataclass
+class PatchMask:
```

Comment:
I'm not sure if this file is the correct place for this class since we do not have a dedicated 'attack utils' file. Should this be moved to its own file?

Reply:
Should be moved to be within `armory.utils`